### DISCUSSION TITLE
lps deck driver build, install

### DISCUSSION THREAD

**Discussion by AlEXANDER9901** [bitcraze/discussions#600] [Q&A]
I'm going to try the loco positiong system. I have a question about lps deck. Assuming I use bitcraze vm, should I build and install lps deck driver? thanks.

**Comment by krichardsson** [comment]
Support for the loco deck is included in the standard build of the firmware and you do not need to do anything special to get started with a standard system. The easiest way is to flash the latest release by following the [Getting started with the Crazyflie 2.X](https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/) tutorial, and then the [Getting started with the Loco Positioning system](https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/) tutorial to set up your Loco system.
